docs/integrations/sources/facebook-marketing.md

```diff
@@ -77,54 +60,110 @@ To set up Facebook Marketing as a source in Airbyte Open Source, you will first
+</FieldAnchor>
+
+<FieldAnchor field="custom_insights.action_breakdowns">
    5. (Optional) For **Action Breakdowns**, use the dropdown list to select the action breakdowns you want to configure.
+</FieldAnchor>
+
+<FieldAnchor field="custom_insights.action_report_time">
    6. (Optional) For **Action Report Time**, enter the action report time you want to configure. This value determines the timing used to report action statistics. For example, if a user sees an ad on Jan 1st but converts on Jan 2nd, this value will determine how the action is reported.
 
       - `impression`: Actions are attributed to the time the ad was viewed (Jan 1st).
       - `conversion`: Actions are attributed to the time the action was taken (Jan 2nd).
       - `mixed`: Click-through actions are attributed to the time the ad was viewed (Jan 1st), and view-through actions are attributed to the time the action was taken (Jan 2nd).
+</FieldAnchor>
 
+<FieldAnchor field="custom_insights.time_increment">
    7. (Optional) For **Time Increment**, you may provide a value in days by which to aggregate statistics. The sync will be chunked into intervals of this size. For example, if you set this value to 7, the sync will be chunked into 7-day intervals. The default value is 1 day.
+</FieldAnchor>
+
+<FieldAnchor field="custom_insights.start_date">
    8. (Optional) For **Start Date**, enter the date in the `YYYY-MM-DDTHH:mm:ssZ` format. The data added on and after this date will be replicated. If this field is left blank, Airbyte will replicate all data.
+</FieldAnchor>
+
+<FieldAnchor field="custom_insights.end_date">
    9. (Optional) For **End Date**, enter the date in the `YYYY-MM-DDTHH:mm:ssZ` format. The data added on and before this date will be replicated. If this field is left blank, Airbyte will replicate the latest data.
-   10. (Optional) For **Custom Insights Lookback Window**, you may set a window in days to revisit data during syncing to capture updated conversion data from the API. Facebook allows for attribution windows of up to 28 days, during which time a conversion can be attributed to an ad. If you have set a custom attribution window in your Facebook account, please set the same value here. Otherwise, you may leave it at the default value of 28. For more information on action attributions, please refer to [the Meta Help Center](https://www.facebook.com/business/help/458681590974355?id=768381033531365).
+</FieldAnchor>
 
-   :::warning
-   Additional data streams for your Facebook Marketing connector are dynamically generated according to the Custom Insights you specify. If you have an existing Facebook Marketing source and you decide to update or remove some of your Custom Insights, you must also adjust the connections that sync to these streams. Specifically, you should either disable these connections or refresh the source schema associated with them to reflect the changes.
-   :::
```

Comment:
Was it intentional to remove this warning?

Reply:
Ah, this part got moved up in the doc, not removed.  I put it alongside the other info about Custom Insights!